<ui version="4.0" >
 <class>ToolboxWidget</class>
 <widget class="QDockWidget" name="ToolboxWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Post Properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents" >
   <layout class="QGridLayout" name="gridLayout_2" >
    <item row="0" column="0" >
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QToolBox" name="toolbox" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="pgBlogs" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>295</width>
           <height>406</height>
          </rect>
         </property>
         <attribute name="label" >
          <string>Blogs</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7" >
          <item row="0" column="0" >
           <widget class="QFrame" name="frameBlog" >
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_6" >
             <item row="0" column="0" >
              <widget class="QRadioButton" name="rbtnBlog1" >
               <property name="text" >
                <string>Blog 1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <spacer name="spaceBlog" >
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>222</width>
                 <height>174</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout_5" >
            <item>
             <widget class="QPushButton" name="btnBlogAdd" >
              <property name="text" >
               <string>&amp;Add...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnBlogEdit" >
              <property name="text" >
               <string>&amp;Edit...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnBlogRemove" >
              <property name="text" >
               <string>&amp;Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pgEntries" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>295</width>
           <height>406</height>
          </rect>
         </property>
         <attribute name="label" >
          <string>Entries</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5" >
          <item row="0" column="0" >
           <widget class="QListView" name="listEntries" />
          </item>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <item>
             <widget class="QPushButton" name="btnEntriesReload" >
              <property name="text" >
               <string>&amp;Reload</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnEntriesCopy" >
              <property name="text" >
               <string>Copy &amp;URL</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pgCat" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>295</width>
           <height>406</height>
          </rect>
         </property>
         <attribute name="label" >
          <string>Categories</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4" >
          <item row="0" column="0" >
           <widget class="QFrame" name="frame_2" >
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4" >
             <item>
              <widget class="QCheckBox" name="chkCatCat1" >
               <property name="text" >
                <string>Cat 1</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2" >
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" >
           <layout class="QVBoxLayout" name="verticalLayout_2" >
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" >
              <item>
               <spacer name="horizontalSpacer" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>38</width>
                  <height>17</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnCatAdd" >
                <property name="text" >
                 <string>Add &amp;Category...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" >
              <item>
               <widget class="QLabel" name="lblCatTags" >
                <property name="text" >
                 <string>&amp;Tags :</string>
                </property>
                <property name="buddy" >
                 <cstring>txtCatTags</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtCatTags" />
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pgMedia" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>295</width>
           <height>406</height>
          </rect>
         </property>
         <attribute name="label" >
          <string>Media files</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="pgOptions" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>295</width>
           <height>406</height>
          </rect>
         </property>
         <attribute name="label" >
          <string>Options</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3" >
          <item row="0" column="0" >
           <widget class="QCheckBox" name="chkOptionsTime" >
            <property name="text" >
             <string>Modify time&amp;stamp</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QDateTimeEdit" name="datetimeOptionstimestamp" />
          </item>
          <item row="2" column="0" >
           <layout class="QVBoxLayout" name="verticalLayout_3" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>&amp;Trackback URLs :</string>
              </property>
              <property name="buddy" >
               <cstring>txtOptionsTrackback</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtOptionsTrackback" />
            </item>
           </layout>
          </item>
          <item row="3" column="0" >
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rbtnBlog1</tabstop>
  <tabstop>btnBlogAdd</tabstop>
  <tabstop>btnBlogEdit</tabstop>
  <tabstop>btnBlogRemove</tabstop>
  <tabstop>listEntries</tabstop>
  <tabstop>btnEntriesReload</tabstop>
  <tabstop>btnEntriesCopy</tabstop>
  <tabstop>chkCatCat1</tabstop>
  <tabstop>btnCatAdd</tabstop>
  <tabstop>txtCatTags</tabstop>
  <tabstop>chkOptionsTime</tabstop>
  <tabstop>datetimeOptionstimestamp</tabstop>
  <tabstop>txtOptionsTrackback</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnBlogAdd</sender>
   <signal>clicked()</signal>
   <receiver>ToolboxWidget</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>64</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel" >
     <x>161</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
